app/tests/spec/views/force_auth.js

```diff
@@ -43,9 +43,14 @@ define(function (require, exports, module) {
       broker = new Broker();
       email = TestHelpers.createEmail();
       formPrefill = new FormPrefill();
-      metrics = new Metrics();
       model = new Backbone.Model();
       notifier = new Notifier();
+      metrics = new Metrics({
+        notifier,
+        sentryMetrics: {
+          captureException () {}
```

Comment:
Not sure I understand the question.

I introduced it in these tests because the metrics object tries to parse the DOM attributes and calls `captureException` if it can't. So I could also have fixed the failing tests by adding appropriate DOM attributes, but defining `sentryMetrics` seemed easier.